.editorconfig

```diff
@@ -12,6 +12,12 @@ end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
 charset = utf-8
+indent_style = space
```

Comment:
You forgot to add those same changes to .dir-local.el and .vimrc.

The things is when we make changes to .dir-locals.el, .vimrc and .editorconfig we have to propagate the changes we make to all three files since we cannot make the assumption that contributors have installed and or are using ( in this case ) EditorConfig plugins for Emacs/VIM.

Reply:
done